rcongui/src/components/SettingsView/admins.js

```diff
@@ -76,7 +70,7 @@ const AddAdminItem = ({
           onAdd(name, playerId, role).then(() => {
             setName("");
             setPlayerId("");
-            setRole("");
+            setRole("spectator");
```

Comment:
While I like this addition, there is one little caveat to this: The available roles are defined by the server. And while most servers will sureley never ever touch this config at all, yet alone delete the spectator role, it _might_ still happen. Hence, we should definitely check that the spectator role actually exists, and if not, select any of the other available roles as default, as a fallback :)

Reply:
Didn't know that the roles are defined in a config. Should be fixed now. 

![player_suggestion](https://github.com/user-attachments/assets/ce0644b2-4263-47b9-b2f5-71a0aabbcabb)